CODEOWNERS

```diff
@@ -9,6 +9,8 @@
 /.known-issues/                           @inakypg @nashif
 /arch/arc/                                @vonhust @ruuddw
 /arch/arm/                                @MaureenHelm @galak
+/arch/arm/core/cortex_m/cmse/             @ioannisg
+/arch/arm/include/cortex_m/cmse/          @ioannisg
```

Comment:
cc: @nashif 
There's no `/arch/arm/include/cortex_m/cmse/` directory , did you mean `/arch/arm/include/cortex_m/cmse.h` ?

As reported by the new and fixed Codeowners check just merged in PR https://github.com/zephyrproject-rtos/ci-tools/pull/54

Reply:
Forgot to explain the urgency of this issue sorry: now anyone trying to add a new file anywhere will experience a failed Codeowners check in github. The "quickfix" is to remove this line; is there a better fix?

Reply:
Hi, sorry for this. Just replace the line as you proposed, above (with cmse.h)